cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
```

Comment:
Since we are using view model to take care of formatting our model to more user friendly way I would recommend moving it to the viewmodel, there is no reason for view controller to take care of keeping this property is another responsibility that makes it harder for future developer to read and work with it